### PR TITLE
Added smartTV guidelines

### PR DESCRIPTION
**Pull request by DenisChinin**
Also fix localStorage polyfill link

### CODE REVIEW

**Comment by ruiposse** [comment]
Hi @DenisChinin thanks for contributing :)

File: README.md [hunk 1]
```diff
@@ -98,9 +98,14 @@ Useful official documentation links about these topics:
 
 ## Design Guidelines
 
-(_TO-DO_)
-
-"Designing for TV" - Developer Android - [http://developer.android.com/training/tv/index.html](http://developer.android.com/training/tv/index.html)
+* [LG Netcast icon & screenshot guide](guidelines/lg_guidelines_icon_and_screenshot_v4.1.pdf)
+* [LG Netcast ui guide](guidelines/lg_smart_tv_and_media_product_ui_guideline_v4_18.pdf)
+* [WebOS UX checklist](http://developer.lge.com/webOSTV/design/ux-checklist/)
```
Comment (ruiposse) [inline README.md]: Do you think we can make both these links external instead of "hosting" the files themselves ?

Comment (DenisChinin) [inline README.md]: Unfortunately no. LG throws error on direct link.  We can provide instruction such as `Netcast TV -> Documentation tab -> Technical Notes -> [NetCast] LG Smart TV and Media Product UI Guideline`   What do you think?

Comment (ruiposse) [inline README.md]: I would very much prefer not to host those documents but links instead. What if we provide links like:
[http://developer.lge.com/resource/tv/RetrieveDocReferencesContent.dev?resourceId=RS00000737](http://developer.lge.com/resource/tv/RetrieveDocReferencesContent.dev?resourceId=RS00000737)

I know it's not the file itself, but at least we're pointing in the right direction without hosting them.
Your thoughts ? :)

**Comment by DenisChinin** [comment]
@ruiposse i've removed pdf guides from git history

**Comment by ruiposse** [comment]
Thanks for the contribution ! :+1:

**Pull request merged**